### PR TITLE
Modification-->Changed the readme and installation files

### LINKED ISSUE #19: MAPBOX_TOKEN missing in .env
## Description

In the Installation Guide, the documentation fails to mention the need to create a MAPBOX_TOKEN in the .env file. Without this token, the server will not start, resulting in an error:
```
Error: Cannot create a client without an access token
    at NodeClient.MapiClient (/path/to/node_modules/@mapbox/mapbox-sdk/lib/classes/mapi-client.js:25:11)
```

**Affected Pages**:
- [ ] README.md
- [x] Installation Guide
- [ ] Configuration Guide
- [ ] Other (please specify)

## Type of Documentation Change

- [x] Update existing documentation
- [ ] Add new documentation
- [ ] Remove outdated or redundant documentation
- [ ] Other (please specify)

## Additional Context
When following the installation steps, users encounter an error due to the missing MAPBOX_TOKEN environment variable. The documentation should explicitly instruct users to include the MAPBOX_TOKEN in the .env file to avoid this issue.

## Suggested Improvements

Update the Installation Guide to include instructions for setting up the .env file, specifically adding:

`MAPBOX_TOKEN=<your_mapbox_token_here>`

This change will ensure that users are aware of the required Mapbox token and avoid runtime errors related to missing environment variables.

---

### PR DESCRIPTION
**Pull request by ARRY7686**
## 📋 Description

Changed the ReadMe by adding info about Mapbox and added a section in Installation about steps on how to obtain environmental variables

## 🔨 Changes Made

- INSTALATION.md --> Added a section
- README.md ---> Added a line about mapbox
- 

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings.
- [x] I have tested it locally and it works fine.
- [x] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [x] Bug fix (non-breaking change which fixes an issue) 🐛
- [ ] New feature (non-breaking change which adds functionality) ✨
- [ ] UI enhancement (non-breaking change which enhances UI) 🎨
- [x] Documentation update 📚

## 🤝 Related Issues

- Fixes #19 

- 

---

## Reminder:
Please ensure your pull request is made to the `test` branch, not `main`.

*Thank you for contributing to VigneshDevHub! We look forward to reviewing your PR and getting it merged!*

### CODE REVIEW

**Comment by ARRY7686** [comment]
@Vignesh025 this is a new PR I am creating pls review the changes and merge it with the code
thank you

File: INSTALLATION.md [hunk 1]
```diff
@@ -17,8 +17,23 @@ To set up the project locally, follow these steps:
...
+    3.2 **Cloudinary**:
+        Go to cloudinary website and sign up there
+        create a new project and obtain the following fields
+        ```
+        CLOUDINARY_CLOUD_NAME=your_cloudinary_cloud_name
+        CLOUDINARY_KEY=your_cloudinary_api_key
+        CLOUDINARY_SECRET=your_cloudinary_api_secret
+        ``` 
+    3.3 **Mapbox**:
+        Go to Mapbox website and sign up there to obtain the mapbox token
```
Comment (Vignesh025) [inline INSTALLATION.md:34]: All the steps are mentioned

File: README.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ The application requires a few configurations to run:
 
 - **MongoDB Atlas**: Set up a cluster and obtain the connection string.
 - **Cloudinary**: Create an account to manage image uploads.
+- **Mapbox**:Create an account and get the MAPBOX TOKEN.
```
Comment (Vignesh025) [inline README.md:54]: Changes also made in README

**Comment by Vignesh025** [comment]
*Thank you for Contributing!*

**Pull request merged**